OpenRA.Mods.Common/Traits/HealsPassengers.cs

```diff
@@ -0,0 +1,82 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Heals all viable passengers simultaneously then optionally unloads fully-healed actors.",
+		"Example use: Hospital structure.")]
```

Comment:
Can you instead showcase the example use? Place a hospital in the TS test map and setup the traits as you imagine them.